compiler-rt/lib/sanitizer_common/sanitizer_platform_interceptors.h

```diff
@@ -523,6 +524,21 @@
 #else
 #define SI_MAC_DEPLOYMENT_BELOW_10_10 0
 #endif
+
+#if defined(__ENVIRONMENT_MAC_OS_X_VERSION_MIN_REQUIRED__) && \
```

Comment:
While these definitions named `SI_MAC_DEPLOYMENT_AT_LEAST_10_XX` are very useful to RADSan, I worry that they might be unwanted maintenance for the LLVM folks. Do you see them having application in other sanitizers? If yes, that's cool - if not, maybe we could move them into the RADSan code?